App.Loans/Models/Loan.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace App.Loans.Models
+{
+    public class Loan
+    {
+        private double money;//денежки
+        public int time { get; set; }//кол-во платежей (ежемесячных)
+        public double money_taked { get; set; }//скок взял
+        public double money_back { get; set; }//скок вернул
+        public double money_left { get { return money; } set { money = money_taked + (money_taked * (time/12) *percent) - money_back; } } //скок осталось вернуть 
```

Comment:
1) you use the same calculation formula in 2 places. You repeat yourself, that means you should this logic into distinct method and reuse it.
2) "money_left" property should be readonly and should not have "set" modifier, because it is calculated from values of "money_taked" and "money_back"